adhocracy4/polls/assets/PollQuestions.jsx

```diff
@@ -183,23 +183,13 @@ class PollQuestions extends React.Component {
 
   sendRequest (datalist) {
     api.poll.batchvote(datalist)
-      .then(respondedQuestions => {
-        respondedQuestions.forEach(rq => {
-          this.setModified(rq.question.id, false)
-          this.setState(prevState => {
-            const questionsCopy = prevState.questions
-            const newQuestions =
-              questionsCopy.map(q => {
-                return rq.question.id === q.id
-                  ? rq.question
-                  : q
-              })
-            return {
-              loading: false,
-              questions: newQuestions,
-              alert: ALERT_SUCCESS
-            }
-          })
+      .then(() => {
+        this.getPollData()
+        this.setState(prevState => {
+          return {
+            loading: false,
+            alert: ALERT_SUCCESS
+          }
         })
```

Comment:
@khamui Hope nothing gets lost here? I needed the updated hasUserVote, so had the get the whole poll data here...